src/components/profile/UpdateProfileForm.jsx

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable camelcase */
```

Comment:
It is important to follow naming conventions. That is why they are added in standard es-linters. Consider removing this line and use standard naming conventions your variables. This helps with readability and maintainability of your code.

Reply:
@Manorlds-Eaglespark, I used the `/* eslint-disable camelcase */` because the names called upon from backend are not of camelcase eg. first_name, last_name, date_of_birth, so that's why I had use of this.